src/cascadia/TerminalApp/Profile.h

```diff
@@ -104,10 +94,11 @@ namespace winrt::TerminalApp::implementation
         GETSET_PROPERTY(hstring, Padding, DEFAULT_PADDING);
 
         GETSET_PROPERTY(hstring, Commandline, L"cmd.exe");
+        GETSET_PROPERTY(hstring, StartingDirectory, L"");

```

Comment:
prefer `{}` or just leave it unfilled (it will get default-constructed)

Remember Niksa always pointing out using empty string literals.